docs/use-keyring-vault-component.md

```diff
@@ -0,0 +1,112 @@
+
+!!! admonition "See also"
+
+    [Hashicorp Documentation: Installing Vault]
+        
+    [Hashicorp Documentation: Production Hardening]
+
+--8<--- "keyring-components-installation.md"
+
+The following example is a global manifest file that does not use local manifests:
+
+```json
+{
+ "read_local_manifest": false,
+ "components": "file:///component_keyring_vault"
+}
+```
+
+The following is an example of a global manifest file that points to a local manifest file:
+
+```json
+{
+ "read_local_manifest": true
+}
+```
+
+The following is an example of a local manifest file:
+
+```json
+{
+ "components": "file:///component_keyring_vault"
+}
+```
+
+The configuration settings are either in a global configuration file or a local configuration file.
+
+The `component_keyring_vault.cnf` file contains the following information:
+
+* `read_local_config [optional]` - this option can be used only in the global configuration file. If you use this option, it indicats whether the component should read configuration information from the local configuration file. The allowed values are `true` or `false`. If you do not use this option, the component uses only the global configuration file.
+
+    If you use the `read_local_config` option in the global configuration file along with other items, the component checks the `read_local_config` item value first:
+
+    `false` - the component processes other items in the global configuration file and ignores the local configuration file.
+
+    `true` - the component ignores other items in the global configuration file and attempts to read the local configuration file.
+
+* `timeout` - the duration in seconds for the Vault server connection timeout. The default value is 15. The allowed range is from 0 to 86400. The timeout can be also disabled to wait an infinite amount of time by setting this variable to 0.
+
+* `vault_url` - the Vault server address.
+
+* `secret_mount_point` - the mount point name where the keyring_vault stores the keys.
+
+* `token` - a token generated by the Vault server.
+
+* `secret_mount_point_version [optional]` - the `KV Secrets Engine version (kv or kv-v2)` used. The allowed values are `AUTO`, `1`, and `2`. The default value is `AUTO`.
+
+* `vault_ca [optional]` - if the machine does not trust the Vault’s CA certificate, this variable points to the CA certificate used to sign the Vault’s certificates.
+
+??? example "Example of a configuration file in JSON format"
+
+     ```json
+     {
+      "timeout": 15,
+      "vault_url": "https://vault.public.com:8202",
+      "secret_mount_point": "secret",
+      "secret_mount_point_version": "AUTO",
+      "token": "58a20c08-8001-fd5f-5192-7498a48eaf20",
+      "vault_ca": "/data/keyring_vault_confs/vault_ca.crt"
+     }
+     ```
+
+!!! warning
```

Comment:
Can we move this to the secret mount point text?

Reply:
The warning is about `secret_mount_point` and the text is about `secret_mount_point_version` parameter